packages/project-access/src/index.ts

```diff
@@ -3,6 +3,7 @@ export {
     findAllApps,
     findProjectRoot,
     getAppRootFromWebappPath,
+    getAppLanguage,
```

Comment:
```suggestion
    getAppProgrammingLanguage,
```

I am wondering if app language could mean natural language. I can live with it though.